ApiFirst/ApiFirst/Customer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace DomainModels.Models
+{
+    public class Customer : BaseModel
+    {
+        public Customer(string fullName, string email, string emailConfirmation, string cpf, string cellphone, DateTime dateOfBirth, bool emailSms, bool whatsapp, string country, string city, string postalCode, string address, string number)
+        {
+            FullName = fullName;
+            Email = email;
+            EmailConfirmation = emailConfirmation;
+            Cpf = cpf.Trim().Replace(".", "").Replace("-", "");
+            Cellphone = cellphone;
+            DateOfBirth = dateOfBirth;
+            EmailSms = emailSms;
+            Whatsapp = whatsapp;
+            Country = country;
+            City = city;
+            PostalCode = postalCode;
+            Address = address;
+            Number = number;
+        }
+
+        public string FullName { get; set; }
+        public string Email { get; set; }
+        public string EmailConfirmation { get; set; }
+        public string Cpf { get; set; }
+        public string Cellphone { get; set; }
+        public DateTime DateOfBirth { get; set; }
+        public bool EmailSms { get; set; }
+        public bool Whatsapp { get; set; }
+        public string Country { get; set; }
+        public string City { get; set; }
+        public string PostalCode { get; set; }
+        public string Address { get; set; }
+        public string Number { get; set; }
```

Comment:
```suggestion
        public int Number { get; set; }
```
Number é do tipo int, conforme descrito no card